Dockerfile

```diff
@@ -44,6 +44,8 @@ RUN \
 ARG FROM
 FROM ${FROM} AS main
 
+COPY docker/unit.list /etc/apt/sources.list.d/unit.list
+ADD --chmod=444 --chown=0:0 https://unit.nginx.org/keys/nginx-keyring.gpg /usr/share/keyrings/nginx-keyring.gpg
```

Comment:
Shouldn't we just add the key to our repo? Then this could be a COPY instead.

Reply:
If we have a copy of the key in our repo it could become outdated and would need to be maintained. With the `ADD` we always get the current key.
But I'm open to changing it.